app/ui-utils/client/lib/messageContext.js

```diff
@@ -27,11 +30,14 @@ export function messageContext({ rid } = Template.instance()) {
 		}),
 		settings: {
 			translateLanguage: AutoTranslate.getLanguage(rid),
+			autoImageLoad: getUserPreference(user, 'saveMobileBandwidth'),
```

Comment:
This references 'saveMobileBandwidth' instead of autoImageLoad. Maybe the problem regarding #15890 ?